### ISSUE TITLE
Player Head Issues

### ISSUE THREAD

**Issue by StellarExile** [KOTORCommunityPatches/K1_Community_Patch#404]
Not sure if this is worth it, but I know for a fact that some player heads have some pretty bad issues. The player head PMHC03 has some deformation and eyelid clipping, and PMHA02 has poorly done eyelids. Above all, PFHA01 has some pretty messy hair. The head model was actually used for Luxa in TSL and was fixed by redrob41 [here](https://deadlystream.com/files/file/452-luxa-hair-fix/) but the hair still remains messy in KOTOR 1.

**Comment by LaIngobernable** [comment]
PMHC03's face deformation is most likely by design, considering the deformed ear and big scar next to it.

**Comment by Salk73** [comment]
In the initial sequence I noticed a problem with the rendering of the teeth.

I'm attaching here four pictures taken in sequence to illustrate.

![295610287-f47d5091-10ea-4825-bba2-123e00b15fde](https://github.com/KOTORCommunityPatches/K1_Community_Patch/assets/49585226/33b41ffb-7142-45d3-b1e3-e7ece75d598a)
![295610328-889feabe-3ab7-414a-b193-c787a7e3c4b6](https://github.com/KOTORCommunityPatches/K1_Community_Patch/assets/49585226/ad0b4a16-7435-40f5-b3d6-0c686b088056)
![295610352-b9ded73c-195e-40ec-9184-7de5ae62ca06](https://github.com/KOTORCommunityPatches/K1_Community_Patch/assets/49585226/d532ecf3-9319-400b-95c1-d53e265ac541)
![295610376-75439328-7114-47c9-85fa-695e1cf6f5b8](https://github.com/KOTORCommunityPatches/K1_Community_Patch/assets/49585226/0f9ad262-71ae-443c-8e9e-399bdc620d50)

Can something be done about it?

**Comment by DarthParametric** [comment]
What body size? That's the midget, right? Presumably it's an issue with the stunt animation, not so much the head itself. You could try testing the head with the other body to confirm. It would also be useful to test some other heads with the midget body to see if they suffer similar issues. At a guess it's probably the tongue clipping through the teeth.

**Comment by Salk73** [comment]
Yeah, body size midget.

I'll run more tests.

**Comment by Salk73** [comment]
Alright.

The issue is there with all body sizes (tested only PFHC04 but I can test other heads as well, if needed) and the stunt animation edit in "Teeth and Tongue All Keys Removed" removes the issue, but only for the midget size.

**Comment by DarthParametric** [comment]
Yeah both versions only edited the keys for CUT015W and CUT016W which are the first two midget shots. You can try this one which removes the keys for teeth and tongue for all anims, but ideally you'd want to test it with all heads to make sure that doesn't introduce other issues.

[K1_Endar_Spire_1st_CS_Stunt_Anim_Edits_v3.zip](https://github.com/KOTORCommunityPatches/K1_Community_Patch/files/13898028/K1_Endar_Spire_1st_CS_Stunt_Anim_Edits_v3.zip)

**Comment by Salk73** [comment]
Will do and report.

Thanks.

**Comment by Salk73** [comment]
I tested 16 different heads and didn't notice any problem.
